src/richie/plugins/lti_consumer/models.py

```diff
@@ -76,15 +93,36 @@ def lti_provider(self):
             self.lti_provider_id, {}
         )
 
+    def get_inline_ratio(self):
+        """
+        Returns plugin "inline_ratio" or defaults to the lti provider setting
+        otherwise returns 0.5625 (16:9 aspect ratio).
+        """
+        inline_ratio = self.inline_ratio
+
+        if inline_ratio is None and self.lti_provider_id:
```

Comment:
That' a use case for the walrus operator! :hand_over_mouth: 
```suggestion
        if inline_ratio:=self.inline_ratio is None and self.lti_provider_id:
```

Reply:
🧠